src/Uno.UI/UI/Xaml/Controls/Border/BorderLayerRenderer.skia.cs

```diff
@@ -74,21 +74,26 @@ internal void Clear()
 		}
 
 		private static IDisposable InnerCreateLayer(
-			ContainerVisual parent,
+			UIElement owner,
 			Rect area,
 			Brush background,
 			Thickness borderThickness,
 			Brush borderBrush,
 			CornerRadius cornerRadius)
 		{
+			var parent = owner.Visual;
 			var disposables = new CompositeDisposable();
 			var subVisuals = new List<Visual>();
 
 			var adjustedLineWidth = borderThickness.Top;
 			var adjustedLineWidthOffset = adjustedLineWidth / 2;
 
-			var adjustedArea = area;
-			adjustedArea.Inflate(-adjustedLineWidthOffset, -adjustedLineWidthOffset);
+			var adjustedArea = new Rect(
```

Comment:
Codacy found an issue: [Remove this useless assignment to local variable 'adjustedArea'.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5924508)